### ISSUE TITLE
creating new cell with "a" is slow

### ISSUE THREAD

**Issue by mmagnuski** [spyder-ide/spyder-notebook#316]
## Description of your problem

**What steps will reproduce the problem?**

1. open Spyder, create empty notebook
2. make sure you are in the command mode
3. press `b` to create a cell below
3. press `a` to create a cell above

**What is the expected output? What do you see instead?**

Creating a cell below is fast (instantaneous), while creating a cell above takes 1 - 2 seconds.
This functionality worked in the previous jupyter-notebook release (`0.2.3`) - but I can't seem to get this previous release working with Spyder 4.

## Versions and main components

* Spyder Version:  4.1.5
* Python Version: 3.7.3
* Qt Version: 5.9.7
* PyQt Version: 5.9.2 (pyqt5-sip=12.8.0)
* Operating system: Windows

### jupyter:
```
# Name                    Version                   Build  Channel
jupyter                   1.0.0                    py37_7
jupyter_client            6.1.6                      py_0
jupyter_console           6.2.0                      py_0
jupyter_core              4.6.3                    py37_0
jupyterlab                2.2.6                      py_0
jupyterlab_pygments       0.1.1                      py_0
jupyterlab_server         1.2.0                      py_0
notebook                  6.1.1                    py37_0
spyder-notebook           0.3.0                      py_0    spyder-ide
```

## Dependencies

```
# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  6.0.6 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.3 (OK)
pygments >=2.0                 :  2.7.1 (OK)
pylint >=1.0                   :  2.6.0 (OK)
pyls >=0.34.0;<1.0.0           :  0.35.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.7 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.2.1 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.2 (OK)

# Optional:
cython >=0.21                  :  0.29.12 (OK)
matplotlib >=2.0.0             :  3.1.0 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  None (OK)

# Spyder plugins:
spyder_notebook                :  0.3.0 (OK)
```

**Comment by jitseniesen** [comment]
Thanks for your report. I can reproduce it, but I have no idea at the moment what the cause of this could be. The issue is not present when running Jupyter Lab directly, so it's either something in the plugin or in the Qt-based web browser.

**Comment by adam-urbanczyk** [comment]
I was able to reproduce it on one system, but on another it works as expected. In the following env everything works quickly:
```
# Name                    Version                   Build  Channel
alabaster                 0.7.13             pyhd8ed1ab_0    conda-forge
anaconda-client           1.11.2             pyhd8ed1ab_0    conda-forge
anaconda-project          0.11.1             pyhd8ed1ab_0    conda-forge
anyio                     3.7.1              pyhd8ed1ab_0    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0           py38h91455d4_3    conda-forge
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
astroid                   2.11.6           py38haa244fe_0    conda-forge
asttokens                 2.0.5              pyhd8ed1ab_0    conda-forge
async-lru                 2.0.4              pyhd8ed1ab_0    conda-forge
atomicwrites              1.4.1              pyhd8ed1ab_0    conda-forge
attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
autopep8                  2.0.2              pyhd8ed1ab_0    conda-forge
babel                     2.12.1             pyhd8ed1ab_1    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bcrypt                    3.2.2            py38h91455d4_1    conda-forge
beautifulsoup4            4.10.0             pyha770c72_0    conda-forge
binaryornot               0.4.4                      py_1    conda-forge
black                     22.1.0             pyhd8ed1ab_0    conda-forge
bleach                    6.0.0              pyhd8ed1ab_0    conda-forge
brotlipy                  0.7.0           py38h294d835_1001    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2023.7.22            h56e8100_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
certifi                   2023.7.22          pyhd8ed1ab_0    conda-forge
cffi                      1.14.5           py38hd8c33c5_0    conda-forge
chardet                   4.0.0            py38haa244fe_1    conda-forge
charset-normalizer        3.2.0              pyhd8ed1ab_0    conda-forge
click                     8.0.3            py38haa244fe_1    conda-forge
cloudpickle               2.2.1              pyhd8ed1ab_0    conda-forge
clyent                    1.2.2                      py_1    conda-forge
cmake                     3.21.3               h39d44d4_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
coloredlogs               15.0.1             pyhd8ed1ab_3    conda-forge
comm                      0.1.4              pyhd8ed1ab_0    conda-forge
conda                     22.11.1          py38haa244fe_1    conda-forge
conda-build               3.23.3           py38haa244fe_1    conda-forge
conda-libmamba-solver     23.1.0             pyhd8ed1ab_0    conda-forge
conda-pack                0.7.0              pyh6c4a22f_0    conda-forge
conda-package-handling    2.0.2              pyh38be061_0    conda-forge
conda-package-streaming   0.7.0              pyhd8ed1ab_1    conda-forge
conda-standalone          22.11.1              h57928b3_0    conda-forge
constructor               3.4.3              pyh95a074a_0    conda-forge
cookiecutter              2.3.0              pyh1a96a4e_0    conda-forge
cryptography              39.0.0           py38h95f5157_0    conda-forge
dataclasses               0.8                pyhc8e2a94_3    conda-forge
debugpy                   1.6.8            py38hd3f51b4_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
diff-match-patch          20230430           pyhd8ed1ab_0    conda-forge
dill                      0.3.7              pyhd8ed1ab_0    conda-forge
docstring-to-markdown     0.12               pyhd8ed1ab_0    conda-forge
docutils                  0.19             py38haa244fe_1    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.3              pyhd8ed1ab_0    conda-forge
executing                 0.8.2              pyhd8ed1ab_0    conda-forge
filelock                  3.3.0              pyhd8ed1ab_0    conda-forge
flake8                    6.0.0              pyhd8ed1ab_0    conda-forge
flit-core                 3.9.0              pyhd8ed1ab_0    conda-forge
fmt                       9.1.0                h181d51b_0    conda-forge
fqdn                      1.5.1              pyhd8ed1ab_0    conda-forge
freetype                  2.12.1               h546665d_1    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
glib                      2.76.4               h12be248_0    conda-forge
glib-tools                2.76.4               h12be248_0    conda-forge
glob2                     0.7                        py_0    conda-forge
gst-plugins-base          1.22.5               h001b923_0    conda-forge
gstreamer                 1.22.5               hb4038d2_0    conda-forge
humanfriendly             10.0             py38haa244fe_4    conda-forge
icu                       72.1                 h63175ca_0    conda-forge
idna                      2.10               pyh9f0ad1d_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        4.10.0           py38haa244fe_0    conda-forge
importlib_metadata        4.10.0               hd8ed1ab_0    conda-forge
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
inflection                0.5.1              pyh9f0ad1d_0    conda-forge
intervaltree              3.1.0              pyhd8ed1ab_1    conda-forge
ipykernel                 6.25.1             pyh6817e22_0    conda-forge
ipython                   8.0.1            py38haa244fe_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
isoduration               20.11.0            pyhd8ed1ab_0    conda-forge
isort                     5.12.0             pyhd8ed1ab_1    conda-forge
jaraco.classes            3.3.0              pyhd8ed1ab_0    conda-forge
jedi                      0.18.1           py38haa244fe_0    conda-forge
jellyfish                 1.0.0            py38h4900a04_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
json5                     0.9.14             pyhd8ed1ab_0    conda-forge
jsonpointer               2.0                        py_0    conda-forge
jsonschema                4.17.3             pyhd8ed1ab_0    conda-forge
jsonschema-with-format-nongpl 4.17.3             pyhd8ed1ab_0    conda-forge
jupyter-lsp               2.2.0              pyhd8ed1ab_0    conda-forge
jupyter_client            8.3.0              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1            py38haa244fe_0    conda-forge
jupyter_events            0.6.3              pyhd8ed1ab_1    conda-forge
jupyter_server            2.7.1              pyhd8ed1ab_0    conda-forge
jupyter_server_terminals  0.4.4              pyhd8ed1ab_1    conda-forge
jupyterlab                4.0.5              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_server         2.24.0             pyhd8ed1ab_0    conda-forge
keyring                   23.9.3           py38haa244fe_1    conda-forge
krb5                      1.20.1               heb0366b_0    conda-forge
lazy-object-proxy         1.9.0            py38h91455d4_0    conda-forge
lcms2                     2.15                 h3e3b177_1    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libarchive                3.6.2                h27c7867_0    conda-forge
libclang                  16.0.1          default_h8b4101f_0    conda-forge
libclang13                16.0.1          default_h45d3cf4_0    conda-forge
libcurl                   7.88.1               h68f0423_1    conda-forge
libdeflate                1.17                 hcfcfb64_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libglib                   2.76.4               he8f3873_0    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libjpeg-turbo             2.1.5.1              hcfcfb64_0    conda-forge
liblief                   0.11.5               h0e60522_0    conda-forge
libmamba                  1.3.1                h8a7d157_2    conda-forge
libmambapy                1.3.1            py38h2899410_2    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libsolv                   0.7.23               h12be248_0    conda-forge
libspatialindex           1.9.3                h39d44d4_4    conda-forge
libsqlite                 3.40.0               hcfcfb64_0    conda-forge
libssh2                   1.10.0               h9a1e1f7_3    conda-forge
libtiff                   4.5.0                hc3b8658_5    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp                   1.3.0                hcfcfb64_0    conda-forge
libwebp-base              1.3.0                hcfcfb64_0    conda-forge
libxcb                    1.13              hcd874cb_1004    conda-forge
libxml2                   2.10.3               hc3477c8_0    conda-forge
libzlib                   1.2.13               hcfcfb64_4    conda-forge
lz4-c                     1.9.3                h8ffe710_0    conda-forge
lzo                       2.10              he774522_1000    conda-forge
m2-msys2-runtime          2.5.0.17080.65c939c               3    conda-forge
m2-patch                  2.7.5                         2    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
m2w64-ntldd-git           15.e7622f6                    3    conda-forge
mamba                     1.3.1            py38hc9c81c8_2    conda-forge
markdown-it-py            2.2.0              pyhd8ed1ab_0    conda-forge
markupsafe                2.0.1            py38h294d835_0    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mccabe                    0.7.0              pyhd8ed1ab_0    conda-forge
mdurl                     0.1.0              pyhd8ed1ab_0    conda-forge
menuinst                  1.4.19           py38haa244fe_1    conda-forge
miniforge_console_shortcut 1.0                  h57928b3_0    conda-forge
mistune                   3.0.1              pyhd8ed1ab_0    conda-forge
more-itertools            10.1.0             pyhd8ed1ab_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
mypy_extensions           0.4.3            py38haa244fe_4    conda-forge
nbclient                  0.8.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.7.4              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.7.4              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.7.4              pyhd8ed1ab_0    conda-forge
nbformat                  5.9.2              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
notebook                  7.0.2              pyhd8ed1ab_0    conda-forge
notebook-shim             0.2.3              pyhd8ed1ab_0    conda-forge
nsis                      3.08                 h2be35fd_1    conda-forge
numpydoc                  1.5.0              pyhd8ed1ab_0    conda-forge
openjpeg                  2.5.0                ha2aaf27_2    conda-forge
openssl                   3.1.2                hcfcfb64_0    conda-forge
overrides                 7.4.0              pyhd8ed1ab_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandoc                    3.1.3                h57928b3_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
paramiko                  3.3.1              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pathspec                  0.9.0              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.4.0            py38h0c7286b_2    conda-forge
pip                       22.3.1             pyhd8ed1ab_0    conda-forge
pkginfo                   1.7.1              pyhd8ed1ab_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
platformdirs              2.5.0              pyhd8ed1ab_0    conda-forge
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
ply                       3.11                       py_1    conda-forge
prometheus_client         0.17.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.24             pyha770c72_0    conda-forge
psutil                    5.8.0            py38h294d835_1    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-lief                   0.11.5           py38h885f38d_0    conda-forge
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pycodestyle               2.10.0             pyhd8ed1ab_0    conda-forge
pycosat                   0.6.3           py38h294d835_1006    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pydocstyle                6.3.0              pyhd8ed1ab_0    conda-forge
pyflakes                  3.0.1              pyhd8ed1ab_0    conda-forge
pygments                  2.11.1             pyhd8ed1ab_0    conda-forge
pylint                    2.14.4             pyhd8ed1ab_0    conda-forge
pylint-venv               3.0.2              pyhd8ed1ab_0    conda-forge
pyls-spyder               0.4.0              pyhd8ed1ab_0    conda-forge
pynacl                    1.5.0            py38hbc1c497_2    conda-forge
pyopenssl                 23.2.0             pyhd8ed1ab_1    conda-forge
pyqt                      5.15.9           py38hd6c051e_4    conda-forge
pyqt5-sip                 12.12.2          py38hd3f51b4_4    conda-forge
pyqtwebengine             5.15.9           py38h30305e3_4    conda-forge
pyreadline3               3.4.1            py38haa244fe_2    conda-forge
pyrsistent                0.18.0           py38h294d835_0    conda-forge
pysocks                   1.7.1            py38haa244fe_3    conda-forge
python                    3.8.15          h4de0772_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.18.0             pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge
python-libarchive-c       3.1              py38haa244fe_0    conda-forge
python-lsp-black          1.2.0              pyhd8ed1ab_0    conda-forge
python-lsp-jsonrpc        1.0.0              pyhd8ed1ab_0    conda-forge
python-lsp-server         1.7.4              pyhd8ed1ab_0    conda-forge
python-lsp-server-base    1.7.4              pyhd8ed1ab_0    conda-forge
python-slugify            8.0.1              pyhd8ed1ab_1    conda-forge
python_abi                3.8                      1_cp38    conda-forge
pytoolconfig              1.2.4              pyhd8ed1ab_0    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pywin32                   300              py38h294d835_0    conda-forge
pywin32-ctypes            0.2.2            py38haa244fe_0    conda-forge
pywinpty                  2.0.11           py38hd3f51b4_0    conda-forge
pyyaml                    5.4.1            py38h294d835_1    conda-forge
pyzmq                     25.1.1           py38ha85f68a_0    conda-forge
qdarkstyle                3.1                pyhd8ed1ab_0    conda-forge
qstylizer                 0.2.2              pyhd8ed1ab_0    conda-forge
qt-main                   5.15.8               h7f2b912_9    conda-forge
qt-webengine              5.15.8               h5b1ea0b_0    conda-forge
qtawesome                 1.2.3              pyhd8ed1ab_0    conda-forge
qtconsole                 5.4.3              pyhd8ed1ab_0    conda-forge
qtconsole-base            5.4.3              pyha770c72_0    conda-forge
qtpy                      2.3.1              pyhd8ed1ab_0    conda-forge
reproc                    14.2.1               h8ffe710_0    conda-forge
reproc-cpp                14.2.1               h0e60522_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
requests-toolbelt         0.10.1             pyhd8ed1ab_0    conda-forge
rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge
rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge
rich                      13.3.1             pyhd8ed1ab_0    conda-forge
ripgrep                   13.0.0               h7f3b576_1    conda-forge
rope                      1.9.0              pyhd8ed1ab_0    conda-forge
rtree                     1.0.1            py38h8b54edf_2    conda-forge
ruamel.yaml               0.17.19          py38h294d835_0    conda-forge
ruamel.yaml.clib          0.2.6            py38h294d835_0    conda-forge
ruamel_yaml               0.15.80         py38h294d835_1004    conda-forge
send2trash                1.8.2              pyh08f2357_0    conda-forge
setuptools                65.6.3             pyhd8ed1ab_0    conda-forge
sip                       6.7.11           py38hd3f51b4_0    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.0.1            py38h32f6830_0    conda-forge
sphinx                    5.1.1              pyhd8ed1ab_1    conda-forge
sphinxcontrib-applehelp   1.0.4              pyhd8ed1ab_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.1              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1              pyhd8ed1ab_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
spyder                    5.4.4            py38haa244fe_0    conda-forge
spyder-kernels            2.4.4           win_pyh7428d3b_0    conda-forge
spyder-notebook           0.5.1              pyh1a96a4e_0    conda-forge
spyder-terminal           1.2.2              pyhd8ed1ab_1    conda-forge
sqlite                    3.34.0               h8ffe710_0    conda-forge
stack_data                0.1.4              pyhd8ed1ab_0    conda-forge
terminado                 0.17.0             pyh08f2357_0    conda-forge
text-unidecode            1.3                        py_0    conda-forge
textdistance              4.5.0              pyhd8ed1ab_0    conda-forge
three-merge               0.1.1              pyh9f0ad1d_0    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tomlkit                   0.12.1             pyha770c72_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.2              py38h91455d4_1    conda-forge
tqdm                      4.57.0             pyhd8ed1ab_0    conda-forge
traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge
typed-ast                 1.5.2            py38h294d835_0    conda-forge
typing-extensions         4.0.1                hd8ed1ab_0    conda-forge
typing_extensions         4.0.1              pyha770c72_0    conda-forge
typing_utils              0.1.0              pyhd8ed1ab_0    conda-forge
typish                    1.9.3              pyhd8ed1ab_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
ujson                     5.7.0            py38hd3f51b4_0    conda-forge
unidecode                 1.3.6              pyhd8ed1ab_0    conda-forge
uri-template              1.3.0              pyhd8ed1ab_0    conda-forge
urllib3                   1.26.15            pyhd8ed1ab_0    conda-forge
vc                        14.2                 hb210afc_3    conda-forge
vc14_runtime              14.34.31931         h5081d32_13    conda-forge
vim                       9.0.0814         py38hd3f51b4_0    conda-forge
vs2015_runtime            14.34.31931         h4c5c07a_10    conda-forge
watchdog                  3.0.0            py38haa244fe_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webcolors                 1.13               pyhd8ed1ab_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.6.1              pyhd8ed1ab_0    conda-forge
whatthepatch              1.0.5              pyhd8ed1ab_0    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
win_inet_pton             1.1.0            py38haa244fe_2    conda-forge
wincertstore              0.2             py38haa244fe_1006    conda-forge
winpty                    0.4.3                         4    conda-forge
wrapt                     1.15.0           py38h91455d4_0    conda-forge
xorg-libxau               1.0.9                hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
yaml                      0.2.5                he774522_0    conda-forge
yaml-cpp                  0.7.0                h63175ca_2    conda-forge
yapf                      0.33.0             pyhd8ed1ab_1    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.6.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               hcfcfb64_4    conda-forge
zstandard                 0.19.0           py38hea8d35e_1    conda-forge
zstd                      1.5.2                h7755175_4    conda-forge
```

**Comment by jitseniesen** [comment]
@adam-urbanczyk Thanks, that is good to know. You said that you have another system where creating a new cell is slow, as in the original bug report. What is the version of spyder-notebook on that system? I had hoped that spyder-notebook version 0.5.1 somehow fixed the issue.

**Comment by adam-urbanczyk** [comment]
It indeed had 0.4. Upgrading to 0.5.1 solved the issue.

**Comment by jitseniesen** [comment]
Thanks, I'll close this issue then as fixed in 0.5.1.
